### PR TITLE
Feature/UI web refactoring

### LINKED ISSUE #13: Refactoring of UI_web to communicate with new API.
UI_web code worked well with the old Banking API, on the other hand, Banking API new version supporting Unit Test and coverage changes the way data is delivered to frontend, which is why refactoring some critical data sharing structures is key for the communication between frontend and backend. 

**Acceptance Criteria**
- 
The following endpoints should be consumed successfully 

1. `api/home`
2. `api/users`
3. `api/login`

### PR DESCRIPTION
**Pull request by Camiloarceg**
Refactoring of the code so it can communicate with the new API

### CODE REVIEW

File: pages/login.py [hunk 1]
```diff
@@ -73,20 +73,25 @@ def login_auth(n_clicks, user, pw, code):
...
         info = json.loads(user_info.text)
-        info_carrier.set_general(info['user'][0])
-        user_type = info_carrier.get_general()
+        print(info)
+        info_carrier.set_general(info)
+        print("hello")
+        print(info_carrier)
+        user_type = info_carrier.get_general()['type']
+        print("hello11")
+        print(user_type)
```
Comment (DMGochoa) [inline pages/login.py]: Shouldn't those prints be in the log or deleted?

File: prueba.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import optparse
+
+opts = optparse.OptionParser()
+
+opts.add_option("-f", "--file", dest='fname', help="This is the file name that you would like to read")
+opts.add_option("-w", "--word", dest='word', help="This is the word that you would like to search")
+
+(options, arguments) = opts.parse_args()
+print(bool(options.fname) == True)
+print(options.word)
```
Comment (DMGochoa) [inline prueba.py]: This file should be deleted

**Review by DMGochoa — CHANGES_REQUESTED** [review]

File: prueba.py (hunk 2 above)
Comment (Camiloarceg) [reply prueba.py]: Fixed,

File: pages/login.py (hunk 1 above)
Comment (Camiloarceg) [reply pages/login.py]: Fixed.

**Review by DMGochoa — APPROVED** [review]
The changes are OK

**Review by calebespinoza — APPROVED** [review]
Tested

**Pull request merged**